data_loader.py

```diff
@@ -264,12 +265,18 @@ def load(self, file_list, cheat_mode, dry_run, loading_mode, wipe_db, max_violat
             if not neo4j_uri:
                 self.log.error('No Neo4j URI specified for backup, abort loading!')
                 sys.exit(1)
-            backup_name = datetime.datetime.today().strftime(DATETIME_FORMAT)
             host = get_host(neo4j_uri)
-            restore_cmd = backup_neo4j(backup_folder, backup_name, host, self.log)
-            if not restore_cmd:
-                self.log.error('Backup Neo4j failed, abort loading!')
-                sys.exit(1)
+            if self.database_type == "neo4j":
+                backup_name = datetime.datetime.today().strftime(DATETIME_FORMAT)
+                restore_cmd = backup_neo4j(backup_folder, backup_name, host, self.log)
+                if not restore_cmd:
+                    self.log.error('Backup Neo4j failed, abort loading!')
+                    sys.exit(1)
+            elif self.database_type == "memgraph":
+                backup_name = backup_memgraph(backup_folder, self.memgraph_snapshot_dir, self.log)
```

Comment:
Instead of having a separate **memgraph_snapshot_dir** can we use the same **backup_folder** used in Neo4j backup?

Reply:
it is using the same backup_folder, the **memgraph_snapshot_dir** is for the location where the memgraph database generate the backup file